ods_ci/tests/Resources/CLI/DSProjects/DSProjects.resource

```diff
@@ -0,0 +1,20 @@
+*** Settings ***
+Documentation    Collection of keywords to interact with the administration of DS Projects via CLI
+
+
+*** Keywords ***
+
+Delete All DS Projects With Name Like
+    [Documentation]    Delete all accelerator profiles with a name that partially or totally matches with the given
```

Comment:
the doc is misleading

Reply:
Fixed in [b1a4d1c](https://github.com/red-hat-data-services/ods-ci/pull/1402/commits/b1a4d1c5afd7abc376bb0f9ef49f64c00be504b0)